### PR TITLE
add text-indent rule for sup

### PR DESCRIPTION
**Pull request by boneskull**
before: before
after: after

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -229,3 +229,8 @@ div.tleft {
 pre, code {
 	font-family: "Latin Modern Mono", monospace !important;
 }
+
+sup {
+	text-indent: 0;
+}
+}
```
Comment (solson) [inline style.css]: Accidental extra `}`?

Comment (boneskull) [inline style.css]: my bad. fixing

**Comment by boneskull** [comment]
arggh sec

**Comment by boneskull** [comment]
alright, fixed.

**Pull request merged**